pkg/stringsearch/fieldnamefinder.go

```diff
@@ -9,20 +9,20 @@ import (
 	"github.com/brimsec/zq/zng/resolver"
 )
 
-type fieldNameFinder struct {
+type FieldNameFinder struct {
```

Comment:
This is fine for now, but package pkg/stringsearch feels like an odd home for something so zng-specific.

Reply:
Agreed.  We do this in a couple places.  I thought about putting it in zq/stringsearch but that didn't feel right either...?

Reply:
I think we should merge this as is and move it later.